nixos/modules/installer/sd-card/sd-image.nix

```diff
@@ -33,6 +32,15 @@ in
   ];
 
   options.sdImage = {
+    makeFsExtraArgs = mkOption {
+      default = {
+        compressImage = true;
+      };
+      description = ''
+        Extra args to pass to make-ext4-fs.nix .
```

Comment:
Maybe link to the file? You'll need to check the right syntax for that though.

Reply:
Indeed, the code is removed now